src/modules/Workspaces/WorkspacesSnapshotTool/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.220531.1" targetFramework="native" />
```

Comment:
using version different than rest of the  solution. Let's change to keep aligned

Reply:
fixed